web/legacy/api/images/get_app_review_images.ts

```diff
@@ -0,0 +1,166 @@
+  req: NextApiRequest,
+  res: NextApiResponse,
+) => {
+  try {
+    if (!process.env.ASSETS_S3_REGION) {
+      throw new Error("AWS Region must be set.");
+    }
+
+    if (!process.env.ASSETS_S3_BUCKET_NAME) {
+      throw new Error("AWS Bucket Name must be set.");
+    }
+
+    if (!protectInternalEndpoint(req, res)) {
+      return;
+    }
+
+    if (req.method !== "GET") {
+      return errorNotAllowed(req.method, res, req);
+    }
+
+    const body = JSON.parse(req.body);
+
+    if (body?.action.name !== "get_app_review_images") {
+      return errorHasuraQuery({
+        res,
+        req,
+        detail: "Invalid action.",
+        code: "invalid_action",
+      });
+    }
+
+    if (body.session_variables["x-hasura-role"] !== "reviewer") {
+      logger.error("Unauthorized access."),
+        { role: body.session_variables["x-hasura-role"] };
+      return errorHasuraQuery({ res, req });
+    }
+
+    const app_id = req.query.app_id;
+    if (!app_id) {
+      return errorHasuraQuery({
+        res,
+        req,
+        detail: "app_id must be set.",
+        code: "required",
+      });
+    }
+
+    // Anchor: Get relative paths for images from the database
+    const client = await getAPIReviewerGraphqlClient();
+
+    const { app: appInfo } = await getAppReviewImages(
+      client,
+    ).GetAppReviewImages({
+      app_id: app_id as string,
+    });
+
+    // If the app is not found, return an error
+    if (appInfo.length === 0 || appInfo[0].app_metadata.length === 0) {
+      return errorHasuraQuery({
+        res,
+        req,
+        detail: "App not found",
+        code: "not_found",
+      });
+    }
+
+    const app = appInfo[0].app_metadata[0];
+
+    const s3Client = new S3Client({
+      region: process.env.ASSETS_S3_REGION,
+    });
+
+    // Anchor: Get Signed URLS for images
+    const objectKey = `unverified/${app_id}/`;
+    const bucketName = process.env.ASSETS_S3_BUCKET_NAME;
+    const urlPromises = [];
+    const command = new GetObjectCommand({
+      Bucket: bucketName,
+      Key: objectKey + app.logo_img_url,
+    });
+
+    if (app.logo_img_url) {
+      urlPromises.push(
+        getSignedUrl(s3Client, command, { expiresIn: 7200 }).then((url) => ({
+          logo_img_url: url,
+        })),
+      );
+    }
+
+    if (app.hero_image_url) {
+      urlPromises.push(
+        getSignedUrl(
+          s3Client,
+          new GetObjectCommand({
+            Bucket: bucketName,
+            Key: objectKey + app.hero_image_url,
+          }),
+          { expiresIn: 7200 },
+        ).then((url) => ({ hero_image_url: url })),
+      );
+    }
+
+    if (app.showcase_img_urls && Array.isArray(app.showcase_img_urls)) {
+      const showcaseUrlPromises = app.showcase_img_urls.map((key) =>
+        getSignedUrl(
+          s3Client,
+          new GetObjectCommand({
+            Bucket: bucketName,
+            Key: objectKey + key,
+          }),
+          { expiresIn: 7200 },
```

Comment:
@andy-t-wang May be it's a good idea to put this expires value under single const